docs/design/proposals/pv-key-rotation.md

```diff
@@ -0,0 +1,55 @@
+# Encryption Key Rotation
```

Comment:
update the file name `rbd-pv-key-rotation` as this is specific to rbd.

Reply:
Ack.